gradle.properties

```diff
@@ -40,7 +40,7 @@ VERSION_NAME=4.1-SNAPSHOT
 # 4.0.0-alpha1 -> 40000031
 # 5.2.3        -> 50203000
 # 5.3.3-beta2  -> 50303072
-VERSION_CODE=40100031
+VERSION_CODE=40100032
```

Comment:
Can we add a comment above this like how we have previously please?

Reply:
Ah, sure. the comments above were always just examples re how to build the VERSION_CODE from a more readable VERSION_NAME. But we might as well use what we do as examples.